extensions/wikia/AdEngine/js/video/videoFrequencyStore.js

```diff
@@ -0,0 +1,103 @@
+/*global define*/
+define('ext.wikia.adEngine.video.videoFrequencyStore', [
```

Comment:
Just wonder how much it is really related to video :) maybe we can call it frequencyStore?